### PR TITLE
🌐 Add Korean translation for `docs/ko/docs/fastapi-cli.md`

### PR DESCRIPTION
**Pull request by dhdld**

### CODE REVIEW

**Review by hard-coders — APPROVED** [review]
LGTM 🚀

**Review by BORA040126 — APPROVED** [review]
LGTM

File: docs/ko/docs/fastapi-cli.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+</div>
+
+`fastapi`라고 불리는 명령어 프로그램은 **FastAPI CLI**입니다.
+
+FastAPI CLI는 Python 프로그램의 경로(예: `main.py`)를 인수로 받아, `FastAPI` 인스턴스(일반적으로 `app`으로 명명)를 자동으로 감지하고 올바른 임포트 과정을 결정한 후 이를 실행합니다.
+
+프로덕션 환경에서는 `fastapi run` 명령어를 사용합니다. 🚀
+
+내부적으로, **FastAPI CLI**는 고성능의, 프로덕션 준비 단계인, ASGI 서버인 <a href="https://www.uvicorn.org" class="external-link" target="_blank">Uvicorn</a>을 사용합니다. 😎
```
Comment (sodaMelon) [inline docs/ko/docs/fastapi-cli.md:63]:
```suggestion
내부적으로, **FastAPI CLI**는 고성능의, 프로덕션에 적합한, ASGI 서버인 <a href="https://www.uvicorn.org" class="external-link" target="_blank">Uvicorn</a>을 사용합니다. 😎
```
프로덕션 준비 단계 -> 프로덕션에 적합한 
i think it's better 
ref : [Define "production-ready"](https://softwareengineering.stackexchange.com/questions/61726/define-production-ready)

File: docs/ko/docs/fastapi-cli.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+## `fastapi dev`
+
+`fastapi dev` 명령을 실행하면 개발 모드가 시작됩니다.
+
+기본적으로 **자동 재시작(auto-reload)** 기능이 활성화되어, 코드에 변경이 생기면 서버를 자동으로 다시 시작합니다. 하지만 이 기능은 리소스를 많이 사용하며, 비활성화했을 때보다 안정성이 떨어질 수 있습니다. 따라서 개발 환경에서만 사용하는 것이 좋습니다. 또한, 서버는 컴퓨터가 자체적으로 통신할 수 있는 IP 주소(`localhost`)인 `127.0.0.1`에서 수신 대기합니다.
```
Comment (sodaMelon) [inline docs/ko/docs/fastapi-cli.md:69]:
```suggestion
기본적으로 **자동 재시작(auto-reload)** 기능이 활성화되어, 코드에 변경이 생기면 서버를 자동으로 다시 시작합니다. 이 기능은 리소스를 많이 사용하며, 비활성화했을 때보다 안정성이 떨어질 수 있습니다. 개발 환경에서만 사용하는 것이 좋습니다. 또한, 서버는 컴퓨터가 자체적으로 통신할 수 있는 IP 주소(`localhost`)인 `127.0.0.1`에서 수신 대기합니다.
```
Removed unnecessary conjunctions.
I think there's a better way to say it than ```수신 대기```.
the wording is clear enough right now.

File: docs/ko/docs/fastapi-cli.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+## `fastapi run`
+
+`fastapi run` 명령을 실행하면 기본적으로 프로덕션 모드로 FastAPI가 시작됩니다.
+
+기본적으로 **자동 재시작(auto-reload)** 기능이 비활성화되어 있습니다. 또한, 사용 가능한 모든 IP 주소인 `0.0.0.0`에서 수신 대기하므로 해당 컴퓨터와 통신할 수 있는 모든 사람이 공개적으로 액세스할 수 있습니다. 이는 일반적으로 컨테이너와 같은 프로덕션 환경에서 실행하는 방법입니다.
+
+대부분의 경우 "종료 프록시(termination proxy)"를 활용해 HTTPS를 처리하는 것이 일반적이며 권장됩니다. 이 설정은 애플리케이션을 어떻게 배포하느냐에 따라 달라질 수 있습니다. 배포 서비스 제공자가 이 작업을 자동으로 처리해줄 수도 있고, 직접 설정해야 할 수도 있습니다.
```
Comment (sodaMelon) [inline docs/ko/docs/fastapi-cli.md:77]:
```suggestion
애플리케이션을 배포하는 방식에 따라 다르지만, 대부분 "종료 프록시(termination proxy)"를 활용해 HTTPS를 처리하는 것이 좋습니다. 배포 서비스 제공자가 이 작업을 대신 처리해줄 수도 있고, 직접 설정해야 할 수도 있습니다.
```
I fixed it to make it clearer and simpler.

There was no ```automatically```(=자동으로) in the original, so I excluded it.

File: docs/ko/docs/fastapi-cli.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+`fastapi dev` 명령을 실행하면 개발 모드가 시작됩니다.
+
+기본적으로 **자동 재시작(auto-reload)** 기능이 활성화되어, 코드에 변경이 생기면 서버를 자동으로 다시 시작합니다. 하지만 이 기능은 리소스를 많이 사용하며, 비활성화했을 때보다 안정성이 떨어질 수 있습니다. 따라서 개발 환경에서만 사용하는 것이 좋습니다. 또한, 서버는 컴퓨터가 자체적으로 통신할 수 있는 IP 주소(`localhost`)인 `127.0.0.1`에서 수신 대기합니다.
+
+## `fastapi run`
+
+`fastapi run` 명령을 실행하면 기본적으로 프로덕션 모드로 FastAPI가 시작됩니다.
+
+기본적으로 **자동 재시작(auto-reload)** 기능이 비활성화되어 있습니다. 또한, 사용 가능한 모든 IP 주소인 `0.0.0.0`에서 수신 대기하므로 해당 컴퓨터와 통신할 수 있는 모든 사람이 공개적으로 액세스할 수 있습니다. 이는 일반적으로 컨테이너와 같은 프로덕션 환경에서 실행하는 방법입니다.
```
Comment (sodaMelon) [inline docs/ko/docs/fastapi-cli.md:75]: Similarly, above commnet, let's re-consider the meaning of ```Listen```.

**Review by sodaMelon — COMMENTED** [review]
hi. thank Thanks for the PR.
i left comments because some sentences are unclear.
명확하지 않은 문장 위주로 제안 드렸습니다. 확인부탁드립니다!

**Comment by dhdld** [comment]
comments 확인하여 명확하지 않은 문장 수정했습니다.
Reviewed the comments and revised unclear sentences.

**Comment by sodaMelon** [comment]
> comments 확인하여 명확하지 않은 문장 수정했습니다. Reviewed the comments and revised unclear sentences. 
LGTM 💖
(수신 대기<-에 관련된 코멘트는 지웠습니다. 저도 주말동안 고민해봤는데 생각이 나질 않네요....)

**Review by sodaMelon — APPROVED** [review]

**Comment by sodaMelon** [comment]
now it's time to merge. 😇

**Comment by alejsdev** [comment]
Thanks for your contribution! @dhdld :tada: 
And thanks for the help @sodaMelon @hard-coders :mag:

**Pull request merged**